log4j-core/src/main/java/org/apache/logging/log4j/core/util/OptionConverter.java

```diff
@@ -297,7 +297,7 @@ public static Object instantiateByClassName(final String className, final Class<
                 if (!superClass.isAssignableFrom(classObj)) {
                     LOGGER.error("A \"{}\" object is not assignable to a \"{}\" variable.", className,
                         superClass.getName());
-                    LOGGER.error("The class \"{}\" was loaded by [{}] whereas object of type [{}] was loaded by [{}].",
+                    LOGGER.error("The class \"{}\" was loaded by [{}] whereas object of type [{}].",
```

Comment:
This message makes no sense. The message will end with "whereas object of type [a.b.C]."  This is only a fragment of  a sentence. I believe the prior format is correct but an argument seems to be missing.

Reply:
> This message makes no sense. The message will end with "whereas object of type [a.b.C]." This is only a fragment of a sentence. I believe the prior format is correct but an argument seems to be missing.

Hi @rgoers 
changed and Jira created
TTY